### ISSUE TITLE
Minus (-) sign next to Tags and Partner organizations confusing

### ISSUE THREAD

**Issue by tbkkr** [research-software-directory/landing-page#75]
I have been wondering for weeks why this minus sign was there. It appears to be a fold-in/out option for the list underneath. Can we change it to the more intuitive V and V (upside down) icons? See
https://www.google.nl/search?q=dropdown+icon&source=lnms&tbm=isch&sa=X&ved=0ahUKEwih6cu549fZAhVKPFAKHW3lDsoQ_AUICigB&biw=1924&bih=937

**Comment by jspaaks** [comment]
Agreed, they are not intuitive. I think at some point they used to say Collapse/Expand which I find even clearer than ``+``, ``-``, ``v`` or ``^`` symbols.
